recipes/nyxus/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "nyxus" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sameeul/nyxus/archive/refs/heads/remove_submodules.zip
+  sha256: 838b33a00ca398e26bc48b172e7dc1a165155587e5a00b4c97ada414a1651fba
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  missing_dso_whitelist:
+    - "*/nvcuda.dll"   # [win]
+
+requirements:
+  build:
+    - cmake
+    - make  # [not win]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - git 
+
+  host:
+    - python
+    - setuptools
+    - pip
+    - z5py 
+    - libtiff 
+    - boost 
+    - nlohmann_json 
+    - blosc 
+    - pybind11 
+    - xtensor 
+    - xsimd 
+    - numpy
+    - pandas 
+    - m2-patch  # [win]
+
+  run:
+    - python
+    - libtiff 
+    - blosc 
+    - {{ pin_compatible('numpy') }}
+    - pandas
+
+test:
+  requires:
+    - pytest
+  imports:
+    - nyxus
+  commands:
+    - python -m pytest run_test.py 
```

Comment:
probably add `pip` to `requires` and `pip check` to `commands`.

Reply:
so, like the following?
```
  requires:
    - pytest
    - pip
  imports:
    - nyxus
  commands:
    - pip check
    - python -m pytest run_test.py
```